packages/jaeger-ui/src/utils/tracking/index.test.js

```diff
@@ -19,11 +19,6 @@ jest.mock('./conv-raven-to-ga', () => () => ({
   message: 'jaeger/a',
 }));
 
-jest.mock('./index', () => {
```

Comment:
This is unrelated but did not find a way around it. The error this produces is https://github.com/facebook/jest/issues/2567 because of the `process` reference. I assume it could be caused be some dependency update but there is no lock file change here and I tried `rm -rf node_modules` and reinstalls multiple times without luck. Anybody knows why this could be happening?